content/docs/user-guide/interactivity/audio/console-variables.md

```diff
@@ -8,32 +8,22 @@ The following console variables can be used with the O3DE audio system.
-| **s_SetRtpc** | Sets an Audio Rtpc value. <br> The first argument is the name of the audio RTPC. <br> The second argument is the float value to be set. <br> The third argument is an optional audio object ID. If the third argument is provided, the RTPC is set on the audio object with the given ID. Otherwise, the RTPC is set on the global audio object. <br> Example: `s_SetRtpc volume_music 1.0` <br> `s_SetRtpc character_speed 0.0 601` |
-| **s_SetSwitchState** | Sets an audio switch to a provided state. <br> The first argument is the name of the audio switch. <br> The second argument is the name of the switch state to be set <br> The third argument is an optional audio object ID. If the third argument is provided, the audio switch is set on the audio object with the given ID; otherwise, the audio switch is set on the global audio object. <br> Example: `s_SetSwitchState weather rain` <br> `s_SetSwitchState SurfaceType concrete 601` |
-| **s_ShowActiveAudioObjectsOnly** | Determines whether only the active audio objects or all audio objects should be drawn when debug drawing is enabled. <br> Default: false (draw all audio objects) |
-| **s_StopTrigger** | Stops an audio trigger. <br> The first argument is the name of the audio trigger to be stopped. <br> The second argument is an optional audio object ID. If the second argument is provided, the audio trigger is stopped on the audio object with the given ID; otherwise, the audio trigger is stopped on the global audio object. <br> Example: `s_StopTrigger MuteDialog` <br> `s_StopTrigger Play_chicken_idle 605` |
-| **s_UnloadPreload** | Unload an Audio Preload from the FileCacheManager. <br> Example: `s_UnloadPreload GlobalBank` |
-| **s_VelocityTrackingThreshold** | An audio object has to change its velocity by at least this amount to issue an `object_speed` RTPC update request to the audio system. <br> Default: 0.1 (10 cm/s) |
+| **s_DrawAudioDebug** | Draws AudioTranslationLayer related debug data to the screen. Flags can be combined. <br> Values: <br><ul><li>0: No audio debug info on the screen</li><li>`a`: Draw spheres around active audio objects</li><li>`b`: Show text labels for active audio objects</li><li>`c`: Show trigger names for active audio objects</li><li>`d`: Show current states for active audio objects</li><li>`e`: Show RTPC values for active audio objects</li><li>`f`: Show Environment amounts for active audio objects</li><li>`g`: Draw occlusion rays</li><li>`h`: Show occlusion ray labels</li><li>`i`: Draw sphere around active audio listener</li><li>`v`: List active Events</li><li>`w`: List active Audio Objects</li><li>`x`: Show FileCache Manager debug info</li><li>`y`: Show memory pool usage info for the audio impl</li></ul> Default: "" (all) <br> Example: `s_DrawAudioDebug abc` |
```

Comment:
If `0` is an actual value that can be used (and not representing the absence of a value), then it should also be formatted.
```suggestion
| **s_DrawAudioDebug** | Draws AudioTranslationLayer related debug data to the screen. Flags can be combined. <br> Values: <br><ul><li>`0`: No audio debug info on the screen</li><li>`a`: Draw spheres around active audio objects</li><li>`b`: Show text labels for active audio objects</li><li>`c`: Show trigger names for active audio objects</li><li>`d`: Show current states for active audio objects</li><li>`e`: Show RTPC values for active audio objects</li><li>`f`: Show Environment amounts for active audio objects</li><li>`g`: Draw occlusion rays</li><li>`h`: Show occlusion ray labels</li><li>`i`: Draw sphere around active audio listener</li><li>`v`: List active Events</li><li>`w`: List active Audio Objects</li><li>`x`: Show FileCache Manager debug info</li><li>`y`: Show memory pool usage info for the audio impl</li></ul> Default: "" (all) <br> Example: `s_DrawAudioDebug abc` |
```

Reply:
Thanks, nice catch!